src/components/DropdownMenuItem/DropdownMenuItem.tsx

```diff
@@ -57,57 +58,55 @@ export interface Props {
  *
  * Dropdown menu item within `DropdownMenu`
  */
-export const DropdownMenuItem = React.forwardRef<HTMLLIElement, Props>(
-  (
-    {
-      align,
-      className,
-      href,
-      children,
-      variant,
-      onClick,
-      status,
-      target,
-      ...other
-    },
-    ref,
-  ) => {
-    const TagName = createTagName();
+export const DropdownMenuItem = ({
+  align,
+  className,
+  href,
+  children,
+  variant,
+  onClick,
+  status,
+  target,
+  ...other
+}: DropdownMenuItemProps) => {
+  const context = useContext(DropdownMenuContext);
+  const ref = (el: HTMLLIElement) => {
```

Comment:
Could you use the full world (ie "element" instead of the truncated version? It makes it a little easier to read.

Reply:
Yup, done